packages/@sanity/form-builder/src/inputs/PortableText/object/BlockObject.styles.ts

```diff
@@ -125,12 +129,16 @@ export const BlockActionsInner = styled(Flex)`
   }
 `
 
-export const TooltipBox = styled(Box)`
-  max-width: 250px;
-`
-export const BlockPreview = styled(Box)((props: {theme: Theme}) => {
-  const color = props.theme.sanity.color.input
+export const StyledButton = styled(Button)((props: {theme: Theme; readOnly: boolean}) => {
   return css`
-    background-color: ${color.default.enabled.bg};
+    --card-bg-color: ${props.readOnly
+      ? props.theme.sanity.color.input.default.disabled.bg
+      : 'inherit'};
+    --card-border-color: 'inherit';
```

Comment:
Should be without quotes. Not on the line above, because it's JS and not CSS!

When we remove the quotes here, an issue is fixed with the focus outline not showing for this button when navigating with the keyboard! 👍